shared/route-tree/index.js

```diff
@@ -1,9 +1,10 @@
 // @flow
```

Comment:
I wrestled with this a bunch and had to use some FlowIssue comments to get this to pass. we can fix it up later